app/bundles/EmailBundle/Tests/Model/SendEmailToContactTest.php

```diff
@@ -725,4 +725,102 @@ function () use ($emailMock) {
         $errorMessages = $model->getErrors();
         $this->assertCount(1, $errorMessages);
     }
+
+    public function testSetSampleMailerByTemporaryMailer()
+    {
+        // Use our test token transport limiting to 1 recipient per queue
+        $transport = new BatchTransport(true, 1);
+        $mailer    = new \Swift_Mailer($transport);
+
+        // Mock factory to ensure that queue mode is handled until MailHelper is refactored completely away from MauticFactory
+        $factoryMock = $this->getMockBuilder(MauticFactory::class)
+            ->disableOriginalConstructor()
+            ->getMock();
```

Comment:
These 3 rows can be simplified into `$factoryMock = $this->createMock(MauticFactory::class);`